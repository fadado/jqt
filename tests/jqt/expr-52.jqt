<!DOCTYPE html>
<html>
  <head>
    <title>{{.variable | @html}}</title>
  </head>
  <body>
{{ .item_list | map("    "+@text) | join(",\n") }}
  </body>
</html>
