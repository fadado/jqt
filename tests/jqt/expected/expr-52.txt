<!DOCTYPE html>
<html>
  <head>
    <title>Value with &lt;unsafe&gt; data</title>
  </head>
  <body>
    1,
    2,
    3,
    4,
    5,
    6
  </body>
</html>

